commit 524239d4b53ba919a18f7b83821d1cf9425a4343
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-06-04T11:50:51-07:00 (4 months ago)
    
    TRITON-1718 sdc-migrate is missing 'abort' help docs
    Reviewed by: Brian Bennett <brian.bennett@joyent.com>
    Approved by: Brian Bennett <brian.bennett@joyent.com>

diff --git a/bin/sdc-migrate b/bin/sdc-migrate
index c3c6a2a..81eb2a6 100755
--- a/bin/sdc-migrate
+++ b/bin/sdc-migrate
@@ -135,6 +135,7 @@ Sub-commands:
     sync VM_UUID                 - sync the filesystems of this migration
     switch VM_UUID               - switch control over to the migrated instance
     pause VM_UUID                - pause a migration operation
+    abort VM_UUID                - abort the migration attempt
     watch VM_UUID                - watch an ongoing migration operation
 
 EOF
@@ -511,6 +512,17 @@ function migration_pause() {
 function migration_abort() {
     local vm_uuid=$1
 
+   # Handle the help argument.
+    if [[ $vm_uuid == "-h" || $vm_uuid == "--help" ]]; then
+        cat <<EOF
+Usage:
+        $0 abort vm_uuid
+
+Aborts the migration, leaving the original instance untouched.
+EOF
+        return 1;
+    fi
+
     [[ -z ${vm_uuid} ]] && fatal "abort: no instance uuid provided"
 
     local url="/vms/${vm_uuid}?action=migrate&migration_action=abort"
diff --git a/package.json b/package.json
index 8654a7b..cf8c3fd 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "sdc",
     "description": "SmartDataCenter 'sdc' zone to hold ops/admin tools",
-    "version": "1.8.0",
+    "version": "1.8.1",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
